src/main/java/hudson/plugins/jobConfigHistory/FileHistoryDao.java

```diff
@@ -25,25 +25,13 @@
 
 import static java.util.logging.Level.FINEST;
 
-import java.io.BufferedOutputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.PrintStream;
+import java.io.*;

```

Comment:
Replace `java.io.*` by the actual required class.

Reply:
Done